examples/config-map.yaml

```diff
@@ -18,6 +18,8 @@ data:
       "ManagementSubnet": "subnet-bad5ede0febe6dd",
       "ClusterName": "aws-virtual-kubelet",
       "Region": "us-west-2",
+      "awsClientTimeoutSeconds":10
+      "AWSClientDialerTimeoutSeconds":2
```

Comment:
How is this different than `AWSClientTimeoutSeconds`?

Reply:
It seems like using https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/custom-http/#timeout covers the Dialer (connect) timeout already.  @SaranBalaji90 is it sufficient to use one timeout that encompasses the entire API call?

> The BuildableHTTPClient can be configured with a request timeout limit. This timeout includes the time to connect, process any redirects, and read the complete response body.

Reply:
@jguice it covers but default timeout is 30s, having less secs gives us ability to retry faster.